### ISSUE TITLE
[Accessibility] Implement aria-checked label on ContextualMenuItem when canCheck is true

### ISSUE THREAD

**Issue by daweffe** [OfficeDev/office-ui-fabric-react#2665]
### Bug Report

- __Package version(s)__: 4.21.3
- __Browser and OS versions__: all

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (No)

Requested priority: (Normal) 

Products/sites affected: Planner

#### Describe the issue:

Screen readers should announce checked/unchecked on ContextualMenuItems when the canCheck property is set to true.  To do this, set the aria-checked label.
